src/commands/create-card.js

```diff
@@ -48,11 +50,9 @@ const handler = async ({ token, column, pullRequest }) => {
         content_type: 'PullRequest'
     };
 
-    try {
-        await createPullRequestCard(inputs);
-    } catch (error) {
-        throw new Error(`Adding a pull request to a project failed. Response: ${error}.`)
-    }
+    const projectCard = await createPullRequestCard(inputs);
```

Comment:
Not sure if I missed something, but what happens with errors? Would they be in the output too?

Reply:
I'm unsure, but we need to look at consistent error handling too. I created an issue for this last week: https://github.com/Financial-Times/github/issues/33.